internal/support/notifications/rest_subscription.go

```diff
@@ -283,6 +303,15 @@ func splitVars(vars string) []string {
 	return strings.Split(vars, ",")
 }
 
+func validateEmail(c models.Channel) bool {
```

Comment:
Can we rename this function to `validateEmailAddresses` for better clarity? Also, could we make this return an `error` rather than a `bool` so that the caller does not have to construct an an `error` like what is happening on lines `107-108`

Reply:
yes, good point, its duplication and there may be future users.

Reply:
Updated the branch, I think validateEmailAddresses has attained its final form, in order to get the errors in I had to make few more changes, which makes it looks cleaner and more in harmony with the rest of the code.